eq_translations/extractable_strings.py

```diff
@@ -12,6 +12,10 @@
         "json_path": "$.sections[*].repeat.title",
         "description": "Section title (repeating section)",
     },
+    {
+        "json_path": "$.sections[*].repeat.page_title",
+        "description": "Section page_title (repeating section)",
```

Comment:
This description is added to the pot file to add context for translators. This should probably be more like "Page title suffix"?